README.md

```diff
@@ -30,13 +32,7 @@ pip install ipydatagrid
 Using `conda`:
 
 ```bash
-conda install ipydatagrid
-```
-
-To enable `ipydatagrid` with JupyterLab:
-
-```bash
-jupyter labextension install @jupyter-widgets/jupyterlab-manager ipydatagrid
+conda install -c conda-forge ipydatagrid
```

Comment:
Is there a conda-forge package already?

Reply:
No conda-forge package yet. Will try to get one out tomorrow! I can remove this reference in the meantime or add a note clarifying the situation

Reply:
I think it is better to remove it until it applies. It's easy to add it back, and especially in these early stages where you have a lot of interest, it will be confusing to have this in here when it isn't true yet.

Reply:
Updated